### DISCUSSION TITLE
I would like to suggest things relevant to enhance the forest website.

### DISCUSSION THREAD

**Discussion by conveyGhost** [ChainSafe/forest#4958] [Ideas]
I had a look at the forest landing page. and it is simple, I like it.
I m a blockchain frontend developer and frontend dev guy,
I d like suggest one for this website.
When I click the each anchor of  global menu, it would be better if it moves smoothly.
![image](https://github.com/user-attachments/assets/936ffe3c-d9f0-4708-9e91-e8ae9e49a5c4)

basically, necessary code is as folow:

```css
html {
 scroll-behavior: smooth;
}
```

I would be pleased if it introduced.
Thank you.
Best luck.

blockchain & Frontend developer conveyGhsota

**Comment by LesnyRumcajs** [comment]
@hanabi1224 Would it be easy to implement from your point of view? Otherwise, @conveyGhost feel free to submit a PR. And thanks for the suggestion!

**Reply by hanabi1224** [reply]
@LesnyRumcajs That sounds good!  @conveyGhost please go ahead submitting a PR if you have time. Thanks!
